1.7/usage/tutorials/cassandra/index.md

```diff
@@ -146,13 +146,13 @@ core@ip-10-0-6-153 ~ $
 You are now inside your DC/OS cluster and can connect to the Cassandra cluster directly. Connect to the cluster using the cqlsh client:
 
 ```bash
-core@ip-10-0-6-153 ~ $ docker run cassandra:2.2.5 cqlsh <HOST>
+core@ip-10-0-6-153 ~ $ docker run -ti cassandra:latest cqlsh <HOST>
```

Comment:
Latest verion: 3.10.
I have made this change i.e. "-ti", as same has been used at other places in the documentation.